travis_configure.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+# Script for setting up OS specific CI test environment on travis-ci.org
+#
+# Copyright (C) 2017 Andreas Bok Andersen <andreas.bok@sociomantic.com>
+#
+
+set -e
+
+# Script which is run before installation script is called
+#
+# Installs fpm needed for deb packaging
+#
+# Returns 0
+before_install()
+{
+    if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+        if [[ "$DIST_PACKAGE_TARGET" == "DEB" ]]; then
+            gem install fpm
```

Comment:
It doesn't. But I will try to add a printout of the gem env variables to see where gems are installed into. Might install it into $HOME/.gems which wouldn't require sudo